plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/StandardProjectWizard.java

```diff
@@ -135,4 +149,31 @@ public void init(IWorkbench workbench, IStructuredSelection selection) {
       }
     }
   }
+
+  private static class DependencyValidator implements IRunnableWithProgress {
+
+    private IStatus result = null;
+
+    @Override
+    public void run(IProgressMonitor monitor) throws InvocationTargetException,
+                                              InterruptedException {
+      BundleContext bundleContext = FrameworkUtil.getBundle(getClass()).getBundleContext();
```

Comment:
Might this be simplified by using the `ServiceContextFactory` to create the `StandardProjectWizard` instance from `com.google.cloud.tools.eclipse.appengine.newproject`'s plugin.xml?